packages/tools/src/tools/annotation/ArrowAnnotateTool.ts

```diff
@@ -446,6 +450,10 @@ class ArrowAnnotateTool extends AnnotationTool {
 
     this.editData = null;
     this.isDrawing = false;
+
+    // This double click was handled and the dialogue was displayed. No need for anyone else to handle it too.
+    evt.stopImmediatePropagation();
```

Comment:
why `stopImmediatePropagation` as opposed to `stopPropagation`? not immediately clear to me so I appreciate an inline comment above it

Reply:
will do